src/main/java/com/snowflake/kafka/connector/internal/streaming/SnowflakeSinkServiceV2.java

```diff
@@ -464,42 +449,6 @@ protected Optional<TopicPartitionChannel> getTopicPartitionChannelFromCacheKey(
   }
 
   // ------ Streaming Ingest Related Functions ------ //
-
-  /* Init Streaming client. If is also used to re-init the client if client was closed before. */
-  private void initStreamingClient() {
```

Comment:
init and close client are essentially copy-pasted into the ingestSdkProvider.java file